.github/workflows/new_theme_bundle.yml

```
@@ -0,0 +1,39 @@
+name: New theme bundle
+
+on:
+  repository_dispatch:
+    types: [new_theme_bundle]
+
+env:
+  release_name: ${{ github.event.client_payload.release }}
+  release_notes_url: ${{ github.event.client_payload.release_notes_url }}
+  bundle_file: ${{ github.event.client_payload.bundle_file }}
+  bundle_url: ${{ github.event.client_payload.bundle_url }}
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup Perl environment
+        uses: shogo82148/actions-setup-perl@v1.15.3
+
+      - name: Update theme bundle in Antora playbook
+        run: |
+          echo "bundle_url=$bundle_url"
+          perl -007 -i -pe 's?.*ui:.*\n.*bundle:.*\n.*url:.*?ui:\n  bundle:\n    url: '$bundle_url'?' antora-playbook.yml
+          cat antora-playbook.yml    
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: "feat(theme_bundle): update theme bundle in Antora playbook"
+          branch: "feat/update_theme_${{ env.release_name }}"
+          delete-branch: true
+          title: "feat(theme_bundle): new theme bundle ${{ env.bundle_file }}"
```

I prefer simpler title. I am not sure `release_name` is related to the version only, but this is what I have in mind.
```suggestion
          title: "feat(theme): update theme bundle to ${{ env.release_name }}"
```

---

ok